images/macos/macos-13-Readme.md

```diff
@@ -145,16 +148,17 @@
 - PowerShell 7.2.11
 
 #### PowerShell Modules
-- Az: 9.7.1
+- Az: 10.0.0
 - Pester: 5.4.1
 - PSScriptAnalyzer: 1.21.0
 
 ### Xcode
-| Version        | Build   | Path                         |
-| -------------- | ------- | ---------------------------- |
-| 14.3           | 14E222b | /Applications/Xcode_14.3.app |
-| 14.2 (default) | 14C18   | /Applications/Xcode_14.2.app |
-| 14.1           | 14B47b  | /Applications/Xcode_14.1.app |
+| Version        | Build   | Path                           |
+| -------------- | ------- | ------------------------------ |
+| 14.3.1         | 14E300b | /Applications/Xcode_14.3.1.app |
```

Comment:
hey the build for `14.3.1` is `14E300c` instead of `14E300b`

Reply:
This is correct. The latest release from Apple has the build number 14E300C.